src/Wire/Internal/NoAllocBitConverter.cs

```diff
@@ -120,5 +120,19 @@ public static unsafe void GetBytes(DateTime dateTime, byte[] bytes)
             }
             bytes[DateTimeSerializer.Size - 1] = (byte) dateTime.Kind;
         }
+        
+        public static unsafe void GetBytes(DateTimeOffset dateTimeOffset, byte[] bytes)
+        {
+            //datetimeoffset size is 11 bytes, ticks + kind + offset seconds
+            fixed (byte* b = bytes)
+            fixed (byte* offset = &bytes[8])
+            {
+                *(long*) b = dateTimeOffset.Ticks;
```

Comment:
Might be a case to go fancy using the new dotnet memory constructs for extra cred :-)

Reply:
That would be cleaner, might be a good refactor for everything later 😃